openedx/core/djangoapps/content/course_views/tabs.py

```diff
@@ -0,0 +1,573 @@
+"""
```

Comment:
I didn't pick `content` as a name. It was already there to support `course_structures` so I thought it might be a natural place to include `course_views` as well. I'm happy to move it somewhere else if need be.